src/ParticipationStorage/ParticipationStorageInterface.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Phpab\Phpab\Storage;
+
+/**
+ * Stores the participation state of the user
+ */
+interface ParticipationStorageInterface
```

Comment:
could be possible that we have AnalyticsStorage later for a e.g. DatabaseAnalytics or LogAnalytics etc. mostly not part in this lib, but could lead to confusion.
This way we are also inline with https://github.com/phpab/phpab/pull/25